docs/docs/step-by-step-walkthrough.md

```diff
@@ -0,0 +1,128 @@
+# Step by Step QHub Cloud Deployment
+
+1) **Environment variables**
+
+    Set the required environment variables based on your choice of provider below:
+
+- [AWS Environment Variables](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/aws/installation.md).
+- [Digital Ocean Environment Variables](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/do/installation.md)
+- [Google Cloud Platform](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/gcp/installation.md)
+
+    After this step, you are ready to initialize `QHub`
+
+2) **Create Cloudflare account**
+    To set up your DNS and automatically get a certificate, you will need to create a [Cloudflare](https://dash.cloudflare.com/sign-up) account and register a [domain name](https://www.cloudflare.com/products/registrar/). 
+
+3) **Initialize QHub**
+    In your terminal, enter the command `qhub init` followed by the abbreviation of your cloud provider. (do: Digital Ocean, aws: Amazon Web Services, gcp: Google Cloud Platform). Example `qhub init do` to initiate QHub for a Digital Ocean deployment. This will create  a `qhub-config.yaml` file in your folder. 
+
+    By default `Qhub` authenticates with github.  Open the config file, and under the `security` section, add your github usernames and set a unique `uid` for each username.
```

Comment:
we should mention that there are other auth options but for simplicity we are using github in this example.